### ISSUE TITLE
Command Line Basics: <Short description of your suggestion>

### ISSUE THREAD

**Issue by johnconway2** [TheOdinProject/curriculum#28323]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the assignment section the hint to use "touch or echo commands" is incorrect based on what we were taught in the software carpentry lesson link. In the software carpentry link in step 1, it has us use the "nano" command to create a new text document. 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-command-line-basics

**Comment by MaoShizhong** [comment]
Thanks for opening this issue.

`nano` opens a new file (unsaved) in the Nano text editor - the file will only be "created" when you save. `touch` just creates the empty file there and then, so it actually exists already. You can then open that file however you wish since it's already been created in that directory.

The software carpentry assigned lessons do mention `touch` as a way to create files, and even discusses the differences between creating via `touch` and `nano`.

Since our lesson assignment wants you to create a new empty file *then* open it in VSCode, it makes sense to use `touch test.txt` then open in VSCode, rather than `nano test.txt` which would then require you to save an empty unsaved file.

I would want to remove the `echo` bit in the hint though. As far as I'm aware, neither our lesson nor the software carpentry bits actually include instructions on creating files with contents via `echo`, and it wouldn't really be appropriate for the task anyway.

Since you specified you didn't want to work on fixing this, I'll open this up for assignment with the following criteria:

**Comment by MaoShizhong** [comment]
## Acceptance criteria

Please comment below if you wish to be assigned to work on this issue. Please do not open a PR until you have been assigned by a maintainer.

- [ ] Remove mention of `echo` from assignment step 3.3's hint (so the hint only mentions the `touch` command)

**Comment by mathdebate09** [comment]
Hey Mao, I could work this fix up :)
